pandas/tests/arrays/categorical/test_repr.py

```diff
@@ -153,8 +153,9 @@ def test_categorical_repr_datetime(self):
             "[2011-01-01 09:00:00, 2011-01-01 10:00:00, 2011-01-01 11:00:00, "
             "2011-01-01 12:00:00, 2011-01-01 13:00:00]\n"
             "Categories (5, datetime64[ns]): [2011-01-01 09:00:00, "
-            "2011-01-01 10:00:00, 2011-01-01 11:00:00,\n"
-            "                                 2011-01-01 12:00:00, "
+            "2011-01-01 10:00:00, 2011-01-01 11:00:00,"
+            "\n                                 "
```

Comment:
Not a big deal, but I think I'd leave the `\n` at the end, seems clearer and more consistent with the spaces at the end